x-pack/test/security_solution_endpoint_configuration/page_objects/index.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { pageObjects as xpackFunctionalPageObjects } from '../../functional/page_objects';
+import { EndpointPageProvider } from '../../security_solution_endpoint/page_objects/endpoint_page';
```

Comment:
Don't really like to import this from other FTR config, but would avoid duplications. Suggestions?

Reply:
If you decide to go with nested folders or two config files then this should be resolved. Also maybe you won't have to duplicate a lot of the other files.